app/src/main/java/org/fossasia/openevent/app/data/models/Ticket.java

```diff
@@ -19,7 +20,7 @@
     private long quantity;
     private String type;
 
-    @ForeignKey(stubbedRelationship = true)
+    @ForeignKey(stubbedRelationship = true, onDelete = ForeignKeyAction.CASCADE)
```

Comment:
I guess I would have got it wrong if it is working. Please continue. I thought it worked through references